src/main/java/seedu/duke/commands/RemoveCommand.java

```diff
@@ -27,6 +27,13 @@ public boolean isInteger (){
         return true;
     }
 
+    boolean isNumberValid(int deleteNumber) {
+        if ( (deleteNumber > 9999) || (deleteNumber < 0) ) {
+            return false;
+        }
+        return true;
+    }
+

```

Comment:
if the maximum size is 9999, you may want to mention in UG, that we only allow max 9999 entries in the list

Reply:
okay! thank you for the suggestions!